keystore/kms/aws/encryptor.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+)
+
+// Configuration represent configuration file structure for AWS KMS
+type Configuration struct {
+	AccessKeyID     string `json:"access_key_id"`
+	SecretAccessKey string `json:"secret_access_key"`
+	Region          string `json:"region"`
+}
+
+// KeyIdentifierPrefix describe prefix used for AWS KMS KeyID
+const KeyIdentifierPrefix = "aws-kms:"
+
+// Encryptor implementation of AWS kms.Encryptor
+type Encryptor struct {
+	client *kms.Client
+}
+
+// NewEncryptor create new AWS KMS encryptor which implement kms.Encryptor interface
+func NewEncryptor(credentialPath string) (*Encryptor, error) {
+	configuration, err := readConfigByPath(credentialPath)
+	if err != nil {
+		return nil, err
+	}
+
+	cfg, err := config.LoadDefaultConfig(context.Background(),
+		config.WithRegion(configuration.Region),
+		config.WithCredentialsProvider(credentials.StaticCredentialsProvider{
+			Value: aws.Credentials{
+				AccessKeyID:     configuration.AccessKeyID,
+				SecretAccessKey: configuration.SecretAccessKey,
+			}},
+		))
+	if err != nil {
+		return nil, err
+	}
+
+	return &Encryptor{
+		client: kms.NewFromConfig(cfg),
+	}, nil
+}
+
+// Source return info about kms BE
+func (e *Encryptor) Source() string {
+	return "KMS AWS"
+}
+
+// Encrypt implementation of kms.Encryptor method
+func (e *Encryptor) Encrypt(keyID string, data []byte) ([]byte, error) {
+	input := &kms.EncryptInput{
+		KeyId:     aws.String(keyID),
+		Plaintext: data,
+	}
+
+	result, err := e.client.Encrypt(context.Background(), input)
```

Comment:
lets use context with at least deadline equal to [default network timeout](https://github.com/cossacklabs/acra/blob/master/network/deadline_listener.go#L27)